sklearn/metrics/scorer.py

```diff
@@ -225,6 +226,10 @@ def get_scorer(scoring):
     """
     if isinstance(scoring, str):
         try:
+            if scoring == 'brier_score_loss':
```

Comment:
please use the method in https://github.com/scikit-learn/scikit-learn/compare/7e079c0fd2%5E...2ba347841